libethereum/Executive.cpp

```diff
@@ -391,6 +400,8 @@ bool Executive::go(OnOpFunc const& _onOp)
 			auto vm = _onOp ? VMFactory::create(VMKind::Interpreter) : VMFactory::create();
 			if (m_isCreation)
 			{
+				if (m_s.addressHasCode(m_newAddress))
```

Comment:
I think this happens only after Metropolis (although I doubt I can trigger this right now).